addons/sale_timesheet/wizard/sale_make_invoice_advance.py

```diff
@@ -28,6 +29,12 @@ def _compute_invoicing_timesheet_enabled(self):
                 )
             )
 
+    @api.depends('sale_order_ids.tasks_ids.timesheet_ids')
```

Comment:
I don't think the `@api.depends` is needed here since we just need to compute the field once we need to read the fields. 🙂